### DISCUSSION TITLE
Version 5.2.x is still in beta so latest docs should be 5.1.3 and not 5.2.x

### DISCUSSION THREAD

**Discussion by gigadie** [twbs/bootstrap#36719] [General] [answered]
Version 5.2.x is still in beta so latest docs should be 5.1.3 and not 5.2.x

Especially cause the api is different substantially in few parts therefore reading the "latest" documentation won't align with what `npm install` will install as latest.

**Comment by mdo** [comment, marked as answer]
Stable is coming this week, sorry for any confusion in the mean time! 💜
